Makefile

```diff
@@ -17,11 +17,11 @@ if_ci_else = $(if $(findstring true,$(CI)),$(1),$(2))
 
 export CENTOS_STREAM_RELEASE = 8
 
-export FEDORA_RELEASE = 38
-export PRIOR_FEDORA_RELEASE = 37
+export FEDORA_RELEASE = 39
+export PRIOR_FEDORA_RELEASE = 38
 
 # This should always be one-greater than $FEDORA_RELEASE (assuming it's actually the latest)
-export RAWHIDE_RELEASE = 39
+export RAWHIDE_RELEASE = 40
 
 # See import_images/README.md
 export FEDORA_IMPORT_IMG_SFX = $(_IMPORT_IMG_SFX)
```

Comment:
oh man...It just occurred to me the AWS builds are probably going to break (or worse, silently use F37/38) unless the import-images process is run.  There are no usable Fedora-beta images in AWS, so they need to be "imported".

Ugggg!  Damn...that's an ugly process and I'm afraid of subjecting you to it.

I mean, you're welcome to try, maybe it will go smoothly?

It's (local) networking intensive, needs a magic key file (in BitWarden), and a devel container (from Makefile).  But the REALLY ugly part is just past all that.  In this really stupid, annoying, asynchronous, and flaky cloud-tool that "converts" images into AMIs (slightly more than just images).  When it breaks, it's impossible to debug and "start over" is about 45 minutes away :sweat: 

Let me see if I can squeeze that in tomorrow morning.  Sorry I totally forgot about this...I think it gave me PTSD and I blocked it out :cursing_face: Sorry man :disappointed_relieved:

Reply:
Yay, got lucky and it worked on the second try.  This small change will allow this PR to continnue:

```diff
diff --git a/IMPORT_IMG_SFX b/IMPORT_IMG_SFX
index 700099f..aa1f708 100644
--- a/IMPORT_IMG_SFX
+++ b/IMPORT_IMG_SFX
@@ -1 +1 @@
-20230816t191121z-f38f37d13
+20230921t135305z-f39f38d13
```